index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <title>Amina Afzal</title>
+  </head>
+  <body>
+    <h1>Amina Afzal</h1>
+    <div class="block">
+      <section class="block_box">
+        <h2>About</h2>
+        <p>
+          My name is Amina Afzal. I was born in Amsterdam, I live in England and my heritage stems back to Rawalpindi, Pakistan. My dream is to be able to work remotely and travel at the same time. In the past I have solo travelled across Japan, Korea, Spain, Turkey and Portugal. I would love to go away for a year and skip from city to city by walking, catching buses or train, cycling even- just not by plane! 
+        </p>
+      </section>
+      <section class="block_box block_photo">
+        <img src="Amina-photo.JPG" alt="Photo of Amina" class="photo" />
+      </section>
+      <section class="block_box">
+        <h2>Careers and Aspirations</h2>
+        <p>To get to my far away dream, I have enrolled onto two bootcamps, one focusing on front-end development and the other in UX/UI design! My goal for the end of the year is to land a full-time tech job and leave the days of pulling coffees behind me. I will miss the free coffees though. </p>
+      </section>
+      <section class="block_box">
+        <h2>Fun Facts!</h2>
+        <p>
+          <ul>
+            <li>I can speak 3 languages, urdu, english and dutch </li>
+            <li>I can move my ears</li>
+            <li>I was born in Amsterdam</li>
+            <li>I can wear contact lenses for 4 days in a row without permanent damage to my eyes </li>
+            <li>I can never go on rollercoasters otherwise the muscle behind my eyeballs could rip </li>    
+          </ul>
+        </p>
+      </section>
+      <section class="block_box block_joke">
+        <h2>Favourite Joke</h2>
+        <p>Knock, knock! <br>
+Who's there? <br>
+Doctor. <br>
+Doctor who? <br>
+Woah really?</p>
+      </section>
+      <section class="block_box block_photo">
+        <h2>Family Tree</h2>
+        <img src="Family-Tree.png" class = "photo"/>
+      </section>
+    </div>
+    <div class ="footer_box">
+      <div class="box1"> <a href="https://www.instagram.com/amina.fz.al"><img src="instagram.png" class="social-media"></a></div>
+      <div class="box2"> <a href="https://twitter.com/aminaxafzal"><img src="twitter.png" class="social-media"></a></div>
+      <div class="box3"> <a href="amina090afzal@gmail.com">amina090afzal@gmail.com</a></div>
+    </div>
+    <footer>
+
+    </footer>
```

Comment:
Might be worth making use of the semantic footer tag here, rather than a div with a separate class. Styles could then be applied directly to the footer.

```suggestion
    <footer>
      <div class="box1"> <a href="https://www.instagram.com/amina.fz.al"><img src="instagram.png" class="social-media"></a></div>
      <div class="box2"> <a href="https://twitter.com/aminaxafzal"><img src="twitter.png" class="social-media"></a></div>
      <div class="box3"> <a href="amina090afzal@gmail.com">amina090afzal@gmail.com</a></div>
    </footer>
```